### PR TITLE
Update utils.py for default logging file encoding config set to "utf-8"

### PR DESCRIPTION
**Pull request by HatuneMiku**
**Proposed changes**:
- ...

**Status (please check what you already did)**:
- [x] made PR ready for code review
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa_nlu#code-style) for instructions)

As #3025

### CODE REVIEW

**Comment by msamogh** [comment]
Thanks for the PR, we'll give it a review as soon as possible.

File: rasa/core/utils.py [hunk 1]
```diff
@@ -36,7 +36,7 @@ def configure_file_logging(logger_obj: logging.Logger, log_file: Optional[Text])
         return
 
     formatter = logging.Formatter("%(asctime)s [%(levelname)-5.5s]  %(message)s")
-    file_handler = logging.FileHandler(log_file)
+    file_handler = logging.FileHandler(log_file, encoding='utf-8')
```
Comment (tabergma) [inline rasa/core/utils.py]:
```suggestion
    file_handler = logging.FileHandler(log_file, encoding="utf-8")
```
Code formatting is failing. You should use double quotes.

**Comment by wochinge** [comment]
@tabergma Can we merge this?

**Comment by tabergma** [comment]
Yes, sorry forgot about it. Will take care of it.

**Pull request merged**